test/users-query.spec.ts

```diff
@@ -0,0 +1,126 @@
+import axios from 'axios';
+import { expect } from 'chai';
+import { PrismaClient, User } from '@prisma/client';
+import { createAuthenticatedSession } from './test-service.js';
+import { seedUsers } from '../scripts/seed-service.js';
+
+const prisma = new PrismaClient();
+
+export const usersQueryTests = (url: string) => {
+  describe('users query tests', () => {
+    beforeEach(async () => {
+      await prisma.user.deleteMany();
+    });
+
+    it('should return users ordered alphabetically with a limit', async () => {
+      const { user, token } = await createAuthenticatedSession(
+        'Jonas Borges',
+        'jonas@teste.com',
+        '2000-01-01',
+        'Test123',
+      );
+      const users = await seedUsers(10);
+      const usersQuery = `
+        query {
+          users(limit: 5) {
+            id
+            name
+            email
+            birthDate
+          }
+        }
+      `;
+
+      const response = await axios.post(
+        url,
+        {
+          query: usersQuery,
+        },
+        {
+          headers: {
+            Authorization: `Bearer ${token}`,
+          },
+        },
+      );
+
```

Comment:
sugestão: utilizando a sintaxe de variáveis do graphql vc conseguiria isolar numa função esse trecho e ia economizar umas boas linhas, fora garantir que todo it usa a mesma query e diminuir o refactor caso tenha a adição/remoção de algum campo na chamada :)